entry.sh

```diff
@@ -2,11 +2,11 @@
 
 echo "balenaBlocks dbus version: $(cat VERSION)"
 
-SESSION_PORT=${DBUS_SESSION_PORT:-55884}
-SYSTEM_PORT=${DBUS_SYSTEM_PORT:-55887}
-
-dbus-daemon --config-file=/usr/src/app/system.conf --fork --address=tcp:host=0.0.0.0,bind=0.0.0.0,port=$SYSTEM_PORT --print-address 
-
-dbus-daemon --config-file=/usr/src/app/session.conf --address=tcp:host=0.0.0.0,bind=0.0.0.0,port=$SESSION_PORT --print-address
-
-balena-idle
\ No newline at end of file
+PORT=${DBUS_PORT:-55884}
+CONFIG=${DBUS_CONFIG:-"session.conf"}
+
+dbus-daemon --config-file=/usr/src/app/$CONFIG \
+	    --fork \
+	    --address=tcp:host=0.0.0.0,bind=0.0.0.0,port=$PORT \
+	    --print-address \
+	&& balena-idle
```

Comment:
not sure if we want to keep the idle here @jakogut 
the container won't restart if there is an err if i'm not mistaken